tests/testthat/_snaps/profile_emissions.md

```diff
@@ -7,15 +7,15 @@
        [3] "country"                            "emission_profile"                  
        [5] "benchmark"                          "ep_product"                        
        [7] "matched_activity_name"              "matched_reference_product"         
-       [9] "co2e_lower"                         "co2e_upper"                        
-      [11] "unit"                               "multi_match"                       
-      [13] "matching_certainty"                 "matching_certainty_company_average"
-      [15] "tilt_sector"                        "tilt_subsector"                    
-      [17] "isic_4digit"                        "isic_4digit_name"                  
-      [19] "company_city"                       "postcode"                          
-      [21] "address"                            "main_activity"                     
-      [23] "activity_uuid_product_uuid"         "profile_ranking"                   
-      [25] "ei_geography"                      
+       [9] "unit"                               "multi_match"                       
+      [11] "matching_certainty"                 "matching_certainty_company_average"
+      [13] "tilt_sector"                        "tilt_subsector"                    
+      [15] "isic_4digit"                        "isic_4digit_name"                  
+      [17] "company_city"                       "postcode"                          
+      [19] "address"                            "main_activity"                     
+      [21] "activity_uuid_product_uuid"         "profile_ranking"                   
+      [23] "ei_geography"                       "co2e_lower"                        
+      [25] "co2e_upper"                        
```

Comment:
The `co2e*` columns moved to the end. We may follow up with an enhancement:

* #224